modules/server_new/src/main/scala/observe/server/tcs/TcsNorth.scala

```diff
@@ -214,8 +216,8 @@ object TcsNorth {
     val offset =
       StepConfig.science.andThen(StepConfig.Science.offset).getOption(stepConfig).map { o =>
         InstrumentOffset(
-          OffsetP(o.p.toAngle.toDoubleDegrees.degrees),
-          OffsetQ(o.q.toAngle.toDoubleDegrees.degrees)
+          OffsetP(Angle.signedDecimalArcseconds.get(o.p.toAngle).toDouble.withUnit[ArcSecond]),
```

Comment:
Maybe provide an `Angle` constructor for `Offset(P|Q)`?

BTW, could we use the `Offset.P` and `Offset.Q` types from `lucuma-core`?

Reply:
I did it but there is an issue, The angles get involved in some calculations that need more precision than what Angle provides, breaking some of the unit test. Since these are values going to the hardware I think it should be accepted thus I reverted to `Quantity[Double, arcSecond]` and it got the unit tests working

Reply:
`Angle` has µarcsec precision.

Reply:
Instrument offsets are very small and they get multiplied for a small factor in process, In the case of an error I had 0.8 mas which was truncated to 0 making it not very good for mathematical operations